api/google_api/util/SceGoogleApiHandler.js

```diff
@@ -325,6 +327,39 @@ class SceGoogleApiHandler {
       });
     });
   }
+  /**
+ * Adds responses from officer app form to a google spreadsheet
+ * @param {String} sheetsID spreadsheet id for the spreadsheet to add to
+ * @param {object} data response data from the officer application form
+ */
+  async writeToForm(sheetsID, data){
+    return new Promise(async (resolve, reject)=>{
+      GoogleSpreadsheet.openById(sheetsID, (error, response) => {
+        if (error){
+          reject(false);
+        }
+      });
+      const doc = new GoogleSpreadsheet(sheetsID);
+      await doc.useServiceAccountAuth(require(this.tokenPath));
```

Comment:
you might want to check if the file exists with `this.checkIfTokenFileExists` before requiring it